Package.swift

```diff
@@ -18,6 +18,7 @@ let package = Package(
     products: [
         .library(name: "ProfileRecorder", targets: ["ProfileRecorder"]),
         .library(name: "ProfileRecorderServer", targets: ["ProfileRecorderServer"]),
+        .library(name: "ProfileRecorderSampleConversion", targets: ["ProfileRecorderSampleConversion"]),
```

Comment:
@vsarunas hmm, that makes `ProfileRecorderSampleConversion` a public product which wasn't the intention at this point. Would you be happy renaming it to `_ProfileRecorderSampleConversion` such that users need to type `import _ProfileRecorderSampleConversion` to make it clear that this isn't fully supported API at this point?

Reply:
@vsarunas this needs a rebase, for some reason I don't have an "Update branch" button to do it for you.

Up to you if you want to implement the above rename target but keep path suggestion or keep it as is

Reply:
Do you want this or keep as is https://github.com/ordo-one/swift-profile-recorder/commit/254410c0a0d0da3759a0c7b979a02040c5930b38 ?